scheduled-jobs/build/poll-for-failed-nightlies/Jenkinsfile

```diff
@@ -131,7 +131,7 @@ node() {
                         if (commonlib.ocpReleaseState[build_version]['release'].isEmpty()) {
                             // For development releases, notify TRT and release artists
                             commonlib.slacklib.to('#forum-release-oversight').say("@release-artists ${msg}")
-                        } else {
+                        } else if (!["4.7", "4.8"].contains(stream_name)) {
```

Comment:
`stream_name` has format 4.y.z-0.nightly. I guess you mean `build_version`?

Reply:
🤦🏼 👍🏼